libs/langchain/langchain/embeddings/jina.py

```diff
@@ -7,69 +6,43 @@
 
 from langchain.utils import get_from_dict_or_env
 
+JINA_API_URL: str = 'https://api.jina.ai/v1/embeddings'
+
 
 class JinaEmbeddings(BaseModel, Embeddings):
     """Jina embedding models."""
 
-    client: Any  #: :meta private:
-
-    model_name: str = "ViT-B-32::openai"
-    """Model name to use."""
-
-    jina_auth_token: Optional[str] = None
-    jina_api_url: str = "https://api.clip.jina.ai/api/v1/models/"
-    request_headers: Optional[dict] = None
+    session: Any  #: :meta private:
+    model_name: str = "jina-embeddings-v2-base-en"
+    jina_api_key: Optional[str] = None
 
     @root_validator()
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that auth token exists in environment."""
         # Set Auth
-        jina_auth_token = get_from_dict_or_env(
-            values, "jina_auth_token", "JINA_AUTH_TOKEN"
+        jina_api_key = get_from_dict_or_env(
+            values, "jina_api_key", "JINA_API_KEY"
```

Comment:
for backwards compatibility should we check for JINA_AUTH_TOKEN as well if JINA_API_KEY isn't found?

Reply:
good idea, I will do

Reply:
I have fixed it @baskaryan, may you check again?